2 table costume.sql

```diff
@@ -0,0 +1,38 @@
+use CostumeDB
+drop table if exists dbo.Costume
+go 
+create table dbo.Costume (
+    CostumeId int not null identity primary key,
+    CustomerFirstName varchar(25) not null,
+        constraint c_Costume_customer_first_name_cannot_be_blank check(CustomerFirstName > ''),
+    CustomerLastName varchar(25) not null,
+        constraint c_Costume_customer_last_name_cannot_be_blank check(CustomerLastName > ''),
+    CostumesBought varchar(100) not null,
+        constraint c_Costume_costumes_bought_must_be_one_of_the_costume_choices 
```

Comment:
So far they are only selling those costumes, but since this may change we wouldn't constrain it.